src/Illuminate/Redis/Lua/LuaScriptBuilder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Illuminate\Redis\Lua;
+
+use Illuminate\Redis\Connections\Connection;
+
+/**
+ * The `LuaScriptBuilder` class represents a builder that constructs and executes Redis Lua scripts.
+ */
+class LuaScriptBuilder
+{
+    /**
+     * The Redis connection instance
+     *
+     * @var Connection
+     */
+    public readonly Connection $connection;
+
+    /**
+     * Constructor method that creates a new LuaScriptBuilder instance.
+     *
+     * @param  Connection  $connection  The Redis connection instance
+     */
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    /**
+     * The static constructor method that's create a new instance of LuaScriptBuilder
+     * @param  Connection  $connection  The Redis connection instance
```

Comment:
Please make sure to add a whiteline between the first sentence and params.

Reply:
Thanks! will do